spring-command-pattern-web/src/main/java/com/idspring/commandpattern/controller/TestingController.java

```diff
@@ -0,0 +1,5 @@
+package com.idspring.commandpattern.controller;
+
+public class TestingController {
+    
+}
```

Comment:
why the class is empty?
```suggestion
}

```